pkg/cortex/cortex.go

```diff
@@ -84,6 +85,7 @@ type Config struct {
 	ConfigStore   config_client.Config                       `yaml:"config_store,omitempty"`
 	Alertmanager  alertmanager.MultitenantAlertmanagerConfig `yaml:"alertmanager,omitempty"`
 	RuntimeConfig runtimeconfig.ManagerConfig                `yaml:"runtime_config,omitempty"`
+	MemberlistKV  memberlist.KVConfig                        `yaml:"memberlist_kv"`
```

Comment:
Do you think it's more clear or confusing if we rename the YAML option from `memberlist_kv` to `memberlist`? The `kv` suffix may not me much clear to the final user.

If we keep `memberlist_kv` then I would suggest to rename the root block to `memberlist_kv_config` in `tools/doc-generator/main.go`.